Core/include/Acts/Propagator/StraightLineStepper.hpp

```diff
@@ -274,7 +274,7 @@ class StraightLineStepper {
   /// @param stype [in] The step size type to be set
   /// @param release [in] Do we release the step size?
   void updateStepSize(State& state, double stepSize,
-                      ConstrainedStep::Type stype = ConstrainedStep::actor,
+                      ConstrainedStep::Type stype = ConstrainedStep::navigator,
```

Comment:
Does this even need a default?

Reply:
Good point I think we should just use the param without default. I will also translate this to the other overload